pages/logout.md

```diff
@@ -17,9 +17,15 @@ private: true
 </div>
 
 <script>
-  window.opener.sessionStorage.clear();
-  window.opener.localStorage.removeItem('hasSession');
-  window.opener.localStorage.removeItem('userFirstName');
-  window.opener.location = '/';
-  window.close();
+  window.sessionStorage.removeItem('authReturnUrl');
+  window.localStorage.removeItem('hasSession');
+  window.localStorage.removeItem('userFirstName');
+
+  var isFullScreenLoginEnabled = window.localStorage.getItem('enableFullScreenLogin');
+  if (isFullScreenLoginEnabled) {
+    window.location = '/';
+  } else {
+    window.opener.location = '/';
+    window.close();
+  }
```

Comment:
This file shouldn't be showing in the diff. Can you try re-syncing your branch with master? Generally, that's done via -

```
git pull
git merge origin/master
```

These commands are run while on your feature branch.

Reply:
I followed your instructions and pushed those changes.
https://github.com/department-of-veterans-affairs/vagov-content/pull/207/files/0883913ea7d0c1283dc8dbaf79c1cb3b481eac1e

Looks like the changes to pages/logout.md are now gone and should be ready to merge into master.